src/main/java/org/jline/utils/AttributedCharSequence.java

```diff
@@ -121,6 +122,13 @@ private static boolean attr(StringBuilder sb, String s, boolean first) {
 
     public abstract AttributedStyle styleAt(int index);
 
+    public int styleCodeAt(int index) {
```

Comment:
I agree.  It follows from AttributedStyle.getStyle being non-public, which I didn't really notice.

Fixed in my local tree.  I'm trying to chase down a re-display bug involving wrapped lines; I want to figure that out before I do another push.